content/en/docs/concepts/security/securing-pods.md

```diff
@@ -0,0 +1,216 @@
+
+- Namespaces enforcing the `privileged` policy (i.e. no restrictions) should set audit and warn labels to `baseline`.
+- Namespaces enforcing the `baseline` policy should set audit and warn labels to `restricted`.
+
+Monitoring these annotations and warnings over time will help you identify workloads that can be migrated to namespaces with stricter policies.
+
+## Examples
+
+### Adding Labels in YAML
+
+This YAML file creates a Namespace `my-secure-namespace` that:
+
+- Blocks any pods that don't satisfy the `baseline` policy requirements.
+- Generates a user-facing warning and adds an audit annotation to any created pod that does not meet the `restricted` policy requirements.
+- Pins the versions of the `baseline` and `restricted` policies to v1.22.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-secure-namespace
+  labels:
+    pod-security.kubernetes.io/enforce: baseline
+    pod-security.kubernetes.io/enforce-version: v1.22
+
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/audit-version: v1.22
+    pod-security.kubernetes.io/warn: restricted
+    pod-security.kubernetes.io/warn-version: v1.22
+```
+
+### Adding Labels to Existing Namespaces with `kubectl label`
+
+{{< note >}}
+When an `enforce` policy (or version) label is added or changed, the admission plugin will test each pod in the namespace against the new policy. Violations are returned to the user as warnings.
+{{< /note >}}
+
+#### Applying to All Namespaces
+
+If you're just getting started with the Pod Security Standards, a suitable first step would be to configure all namespaces as `privileged` but set up audit annotations for a stricter level such as `baseline`:
+
+```shell
+kubectl label --overwrite ns --all \
+  pod-security.kubernetes.io/enforce=privileged \
+  pod-security.kubernetes.io/audit=baseline \
+  pod-security.kubernetes.io/warn=baseline
+```
+
+#### Applying to a Single Namespace
+
+You can update a specific namespace as well. This command adds the `enforce=restricted` policy to `my-existing-namespace`, pinning the restricted policy version to v1.22.
+
+```shell
+kubectl label --overwrite ns my-existing-namespace \
+  pod-security.kubernetes.io/enforce=restricted \
+  pod-security.kubernetes.io/enforce-version=v1.22
+```
+
+## Migrating from Pod Security Policies
+
+Migrating to the replacement policy from PodSecurityPolicies can be done effectively using a combination of dry-run and `audit` and `warn` modes, although this becomes harder if mutating PSPs are used).
+
+{{< note >}}
+**We are working on tooling to automate this migration.** In the meantime, this guide should give you an idea of the general process.
+{{< /note >}}
```

Comment:
Whilst this is true, we'd say this in a post-release blog article. In the docs we mostly [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).